pom.xml

```diff
@@ -321,7 +322,17 @@
                             <referenceVersion>5.23ea0</referenceVersion>
                             <artifactsURI>https://teamcity.chronicle.software/repository/download</artifactsURI>
                             <!-- Reduce the API surface of ChronicleReader -->
-                            <binaryCompatibilityPercentageRequired>86.1</binaryCompatibilityPercentageRequired>
+                            <binaryCompatibilityPercentageRequired>89.3</binaryCompatibilityPercentageRequired>
+                            <extraOptions>
+                                <!--This skips "internal" but checks "impl"-->
+                                <extraOption>
+                                    <name>-keep-internal</name>
+                                </extraOption>
+                                <extraOption>
+                                    <name>skip-internal-packages</name>
+                                    <value>.*internal.*</value>
```

Comment:
Should this have a hyphen in front of it too? `-keep-internal` and `-skip-internal-packages` both seem to have hyphens in `japi-compliance-checker -help`

```
  -keep-internal
      Do not skip checking of these packages:
        *impl*
        *internal*
        *examples*
  
  -skip-internal-packages PATTERN
      Do not check packages matched by the regular expression.
  
  -skip-internal-types PATTERN
      Do not check types (classes and interfaces) matched by the regular
      expression. It's matched against full qualified type names (e.g.
      'org.xyz.Name<T>'). It has to match any part of type name.
```

Reply:
Also should this config be in pluginManagement in java-parent-pom perhaps?

Reply:
No `-` in the beginning.